doc/install.md

```diff
@@ -24,7 +24,11 @@ You can also build the executables from scratch.
 
 &ensp; 4\. You need to install `ffmpeg` as a dependency.  Run `./install_ffmpeg.sh`.  This will install the dependencies in `~/compiled`.  You need to have `pkg-config` installed.
 
-&ensp; 5\. You can now run `PKG_CONFIG_PATH=~/compiled/lib/pkgconfig go build ./cmd/livepeer/livepeer.go` from the project root directory. To get latest version, `git pull` from the project root directory.
+&ensp; 5\. You can now run `make HIGHEST_CHAIN_TAG=mainnet PKG_CONFIG_PATH=~/compiled/lib/pkgconfig` from the project root directory. To get latest version, `git pull` from the project root directory.
+
+&ensp; &ensp; a\. `HIGHEST_CHAIN_TAG` represents the highest compatible chain to build for with an ordering of `dev < rinkeby < mainnet`, e.g. using `HIGHEST_CHAIN_TAG=rinkeby` will allow the node to run on the public rinkeby test network but throw an error when trying to connect to the main ethereum network.
+
+&ensp; &ensp; b\. `PKG_CONFIG_PATH` is the path where `pkg-config` from _step 3_ has been installed.
```

Comment:
Technically `PKG_CONFIG_PATH` is the path where the `pkg-config` files for `ffmpeg` dependences live.